### PR TITLE
Added behat

### PR DESCRIPTION
**Pull request by jhryniuk**

### CODE REVIEW

File: app/api/src/Security/Voter/UserVoter.php [hunk 1]
```diff
@@ -10,22 +11,29 @@
...
     {
-        return in_array($attribute, ['USER_READ', 'USER_EDIT', 'USER_DELETE'])
-            && $subject instanceof \App\Entity\User;
+        return (in_array($attribute, ['USER_READ', 'USER_EDIT', 'USER_DELETE'])
+            && $subject instanceof \App\Entity\User) || $this->kernel->getEnvironment() === 'test';
     }
 
     protected function voteOnAttribute(string $attribute, mixed $subject, TokenInterface $token): bool
     {
+        if ($this->kernel->getEnvironment() === 'test') {
```
Comment (dominikpgs) [inline app/api/src/Security/Voter/UserVoter.php]: yoda

File: app/api/src/Security/Voter/UserVoter.php [hunk 2]
```diff
@@ -10,22 +11,29 @@
...
     {
         $this->security = $security;
+        $this->kernel = $kernel;
     }
     protected function supports(string $attribute, mixed $subject): bool
     {
-        return in_array($attribute, ['USER_READ', 'USER_EDIT', 'USER_DELETE'])
-            && $subject instanceof \App\Entity\User;
+        return (in_array($attribute, ['USER_READ', 'USER_EDIT', 'USER_DELETE'])
+            && $subject instanceof \App\Entity\User) || $this->kernel->getEnvironment() === 'test';
```
Comment (dominikpgs) [inline app/api/src/Security/Voter/UserVoter.php]: move `\App\Entity\User` to use

Comment (dominikpgs) [inline app/api/src/Security/Voter/UserVoter.php]: instead
$this->kernel->getEnvironment() === 'test'
consider
'test' === $this->kernel->getEnvironment()

Comment (dominikpgs) [inline app/api/src/Security/Voter/UserVoter.php]: Isn't there const for "test"

File: app/api/src/Security/Voter/UserVoter.php [hunk 3]
```diff
@@ -10,22 +11,29 @@
 class UserVoter extends Voter
 {
     private $security = null;
+    private $kernel = null;
 
     /**
      * @param Security $security
+     * @param KernelInterface $kernel
```
Comment (dominikpgs) [inline app/api/src/Security/Voter/UserVoter.php]: Instead of phpdoc use strist type declaration

File: app/api/src/Security/Voter/UserVoter.php [hunk 4]
```diff
@@ -10,22 +11,29 @@
 class UserVoter extends Voter
 {
     private $security = null;
+    private $kernel = null;
```
Comment (dominikpgs) [inline app/api/src/Security/Voter/UserVoter.php]: $this->kernel cant be null! must be KernelInterface

File: app/api/src/Security/Voter/ImageVoter.php [hunk 5]
```diff
@@ -0,0 +1,44 @@
+<?php
+
```
Comment (dominikpgs) [inline app/api/src/Security/Voter/ImageVoter.php:2]: declare strict type everywhere

File: app/api/src/Entity/User.php [hunk 6]
```diff
@@ -145,7 +145,9 @@ public function getEventsOwner(): Collection
     public function setEventsOwner(array $events): void
     {
         if (!empty($events)) {
-            $this->eventsOwner = $events;
+            foreach ($events as $event) {
```
Comment (dominikpgs) [inline app/api/src/Entity/User.php]:
```
if (empty($events)) {
return;
}
             foreach ($events as $event) {
                 $this->eventsOwner->add($event);
             }
         
         ```

**Review by dominikpgs — CHANGES_REQUESTED** [review]

File: app/api/src/Security/Voter/UserVoter.php (hunk 2 above)
Comment (jhryniuk) [reply app/api/src/Security/Voter/UserVoter.php]: Don't understand the question.

Comment (jhryniuk) [reply app/api/src/Security/Voter/UserVoter.php]: I didn't found const with content 'test' in symfony.

**Pull request merged**